samples/nrf9160/modem_shell/README.rst

```diff
@@ -344,6 +344,44 @@ Examples
 
 ----
 
+Locaton tool
+============
+
+MoSh command: ``location``
+
+You can use the Location tool for retrieving devices location with different location methods.
+See :ref:`lib_location` library for information on the configuration of different location methods and services
+but there are some available by default to ease trials.
+
+Examples
+--------
+
+* Retrieve location with default configuration:
+
+  .. code-block:: console
+
+     location get
+
+* Retrieve location with Wi-Fi positioning (you need to have Wi-Fi build and chip) and if not found, use cellular positioning:
```

Comment:
```suggestion
* Retrieve location with Wi-Fi positioning (you need to have a Wi-Fi-enabled device) and if not found, use cellular positioning:
```

Reply:
This changes the meaning. I'm trying to say that you need to have the device as well as MOSH built with Wi-Fi support

Reply:
How about something like: "You need to have a Wi-Fi enabled device and build the sample Wi-Fi support."

Reply:
That's fine although a bit long in the parenthesis. Can you modify the suggestion?

Reply:
Did that.